doc/user_manual/chIntroduction.tex

```diff
@@ -103,15 +103,15 @@ \section{Evolution of SixTrack}
     \item \textbf{Version 4}
         \todo[inline]{Add stuff}
     \item \textbf{Version 5}
-        Includes a full rewrite of the input parsing, upgrade of the source to Fortran 2008, and makes all major arrays dynamically allocatable.
-        Machine size and particle numbers are no longer defined by compiler flags.
-        Replaces the astuce preprocessor with the c preprocessor.
+        Machine size and particle numbers are no longer defined by compiler flags but allocated dynamically based on the input files.
         Merges the Collimation version of SixTrack into the main code.
-        Code wide implementation of rounding of input float values from text, as well as output of float values to text in critical parts, provided by the CRLIBM library.
+        Code-wide implementation of rounding of input float values from text, as well as output of float values to text in critical parts, provided by the CRLIBM library.
         This also includes a wrapper for consistent rounding of values from mathematical functions.
         Adds experimental support for HDF5 and ROOT output files.
         Extensive rewrite of many sections of the code to increase flexibility and modularity, as well as bug fixes to the physics.
+        Includes a full rewrite of the input parsing and upgrade of the source to Fortran 2008.
         Support for 32, 64 and 128 bit floating point precision.
+        Replaces the astuce preprocessor with the c preprocessor.
```

Comment:
+ split into multiple, smaller files.

Reply:
Too much detail for a summary in my opinion. No?